package.json

```diff
@@ -61,6 +61,7 @@
     "keypress": "^0.2.1",
     "mocha": "^10.0.0",
     "nimble": "^0.0.2",
-    "nyc": "^15.1.0"
+    "nyc": "^15.1.0",
+    "ip": "^2.0.1"

```

Comment:
Hi! Why do you need an `ip` package in the dependencies?

Reply:
When I ran the development environment (`npm run mockserver`) for the tests, this was not found on my system so I included it in the dev dependencies. We can remove it if you didn't want to include it, but I had to manually run `npm install ip --save-dev`.

Reply:
<img width="951" alt="Screenshot 2024-10-27 at 9 11 33 AM" src="https://github.com/user-attachments/assets/3b03f4f3-b52d-486d-804a-8f1f2a68cf27">

Reply:
Oh, that was 5 years ago :moyai:... and there is no need for that now :smile:
I'll merge your code right now and cleanup this part of the code later by myself

Reply:
Sounds good! Thanks for checking into this.